xml/System.ServiceModel.Federation/WSTrustChannelSecurityTokenProvider.xml

```diff
@@ -1,887 +1,887 @@
+      <MemberSignature Language="VB.NET" Value="Public Overrides ReadOnly Property SupportsTokenRenewal As Boolean" />
+      <MemberSignature Language="F#" Value="member this.SupportsTokenRenewal : bool" Usage="System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider.SupportsTokenRenewal" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; virtual property bool SupportsTokenRenewal { bool get(); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>4.8.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Boolean</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <value>To be added.</value>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Abort">
+      <MemberSignature Language="C#" Value="void ICommunicationObject.Abort ();" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance void System.ServiceModel.ICommunicationObject.Abort() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider.System#ServiceModel#ICommunicationObject#Abort" />
+      <MemberSignature Language="VB.NET" Value="Sub Abort () Implements ICommunicationObject.Abort" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit&#xA;override this.System.ServiceModel.ICommunicationObject.Abort : unit -&gt; unit" Usage="wSTrustChannelSecurityTokenProvider.System.ServiceModel.ICommunicationObject.Abort " />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.ICommunicationObject.Abort() = System::ServiceModel::ICommunicationObject::Abort;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.ICommunicationObject.Abort</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Causes a communication object to transition immediately from its current state into the closed state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```